### DISCUSSION TITLE
Connection issues between Boost Hub and Powered Up remote

### DISCUSSION THREAD

**Discussion by vjchan99** [pybricks/support#1112] [General]
Hi,

I am new to pybricks and excited about being able to autonomously connect and control the Boost hub with the Powered Up remote. But I am having difficulty making the connection by just pressing the green buttons on the hub and remote at the same time. What am I doing wrong?

Also, if I send pybrick code to the hub and want to use the remote to control the hub, at which point do I have to connect the remote to the hub? Do I connect them together before sending the code to the hub or after?

Thanks.

**Comment by dlech** [comment]
If you are using Pybricks, you have to write a program to use the remote. The connection takes place when the program starts and ends when the program ends.

You can read more about it at https://pybricks.com/projects/tutorials/wireless/remote-control/button-basics/

**Reply by vjchan99** [reply]
Thanks so much for your help. I have worked it out now.
